dist/commands/head.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var interfacer = require('./../util/interfacer');
+var fs = require('fs');
+var fsAutocomplete = require('vorpal-autocomplete-fs');
+
+var head = {
+  exec: function exec(args, options) {
+    var self = this;
+    options = options || {};
+
+    options.argsType = args.stdin === undefined ? 'files' : 'stdin';
+    options.n = options.n === undefined ? 10 : options.n;
+
+    if (options.n < 1) {
+      self.log('Option n must be a positive integer.');
```

Comment:
@nfischer [Head | IEEE 1003.1](http://pubs.opengroup.org/onlinepubs/009604599/utilities/head.html)

> The head utility shall conform to the Base Definitions volume of IEEE Std 1003.1-2001, Section 12.2, Utility Syntax Guidelines.

> The following option shall be supported:

> -n  number
The first number lines of each input file shall be copied to standard output. The application shall ensure that the number option-argument is a positive decimal integer.
When a file contains less than number lines, it shall be copied to standard output in its entirety. This shall not be an error.

> If no options are specified, head shall act as if -n 10 had been specified.

I see your point here, but to be posix compliant we need to make sure it is a positive decimal integer. I'm not sure about the 0. Is 0 a positive decimal integer?